app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md

```diff
@@ -48,81 +55,74 @@ keywords:
+## Salary progression
 
+Qualified teachers get a minimum starting salary of $salaries_qualifiedteacher_englandmin$ (or more in London). Your salary will be reviewed every year, with most teachers moving up the pay range annually. This will depend on your school’s performance management arrangements. 
 
-As well as helping you understand the demands of a more senior role, a teaching and learning responsibility (TLR) payment could make a big difference to your pay. You could earn up to an extra $salaries_tlr_max$ on top of your salary.
+A typical teacher could earn at least $salaries_fiveyears_max$ (or more in London) after 5 years.  
 
-## What steps do you need to consider if you want to be a headteacher?
+[Find out more about teacher pay](/life-as-a-teacher/pay-and-benefits/teacher-pay). 
 
-<strong>Chris:</strong> If you want to go into senior leadership in a secondary school you’ll need to build both a curriculum role and a pastoral role, for example, head of geography and head of year into your early career.
+### Additional payments
 
-<strong>Elaine:</strong> I've got curriculum and pastoral experience and I also did school timetabling. As a headteacher you need experience of everything to get to the top.
+You may also receive additional payments on top of your regular salary for taking on extra responsibilities. These are known as teaching and learning responsibility (TLR) payments. 
 
-## How long does it take to go from an early career teacher to a headteacher?
+This could be a one-off payment for a specific project, for example, creating a timetable. Or it might be an extra responsibility attached to your job role, for example, being the head of department. 
 
-<strong>Sian:</strong> I think 8 to 10 years is realistic to become a headteacher for primary schools although you may be able to climb the career ladder a bit quicker.
+Depending on your responsibilities, you could earn up to an extra $salaries_tlr_max$. Your school will determine how much you receive.
 
-<strong>Elaine:</strong> If you’re driven you could probably get to headship in between 9 to 12 years. It took me longer but at the time it suited me to stay where I was.
+## National professional qualifications
 
-<strong>Chris:</strong> If you want to develop your leadership skills there are national professional qualification (NPQ) leadership courses available to help teachers progress their careers.
+National professional qualifications (NPQs) are professional development courses for teachers and leaders to develop skills, progress careers and improve school outcomes.  
 
-## How has teaching helped your professional development?
+There are 2 types of NPQs: specialist and leadership. Specialist NPQs can help you develop your classroom teaching or specialism. Leadership NPQs can help you develop your leadership skills or to progress to a leadership role.
```

Comment:
Not sure if it is worth mentioning funding - there is a small para at the end of [the GOV.UK page](https://www.gov.uk/guidance/national-professional-qualification-npq-courses) on this (but that might be too much detail)

Would be good to review this with the CPD team